es-ds-components/components/es-pagination.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <Paginator
+        v-bind="$attrs"
```

Comment:
Another nit: v-bind attrs is unnecessary [because there's only one root element](https://vuejs.org/guide/components/attrs#attribute-inheritance)

Reply:
ah good point